source/content.css

```
@@ -1119,6 +1119,13 @@ body > .footer li a {
 	text-align: center;
 }
 
+/* Enable word wrap on code in comments (with line numbers only) */
+.discussion-item-body .blob-num ~ .blob-code-inner, /* Embedded code */
+.discussion-item-body .blob-num ~ .blob-code .blob-code-inner { /* Review diffs and suggestions */
```

🥇  Lines above should be wrapped.
```suggestion
.wrap-it-all .discussion-item-body .blob-num ~ .blob-code .blob-code-inner { /* Review diffs and suggestions */
```
🥈 Lines above should be wrapped.

https://github.com/sindresorhus/refined-github/blob/b48f289f835fa4447743fe0541dcc3d84c46b063/source/content.css#L358-L359

🥉 Lines above should be wrapped.